package.json

```diff
@@ -58,9 +58,10 @@
     "python:win32": "bash -c 'tasks/pythonInstall.sh'",
     "report": "node e2e/create-report.js",
     "start": "node ./tasks/start.js",
-    "test": "npm run test:unit && npm run test:lint",
+    "test": "npm run test:unit && npm run test:unit:tz && npm run test:lint",
     "test:lint": "npx eslint .",
     "test:unit": "npx jest",
+    "test:unit:tz": "TZ=Australia/Sydney npx jest",
```

Comment:
Looks like this command could be the following to work on Windows:
`"test:unit:tz": "bash -c 'TZ=Australia/Sydney npx jest'",`

Reply:
That doesn't work on Windows. In bash, output should look like this:
```bash
$ node -e 'console.log(new Date(2018, 02, 29))'
2018-03-29T04:00:00.000Z

$ TZ=Australia/Sydney node -e 'console.log(new Date(2018, 02, 29))'
2018-03-28T13:00:00.000Z
```

It can be changed with `tzutil` but that changes it for the whole system and is probably not desirable in an automated fashion.